test/unit/cute/CMakeLists.txt

```diff
@@ -26,31 +26,49 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-add_subdirectory(core)
-add_subdirectory(volta)
-add_subdirectory(ampere)
-add_subdirectory(hopper)
 add_subdirectory(layout)
-add_subdirectory(msvc_compilation)
-
-add_custom_target(
-  cutlass_test_unit_cute
-  DEPENDS
-  cutlass_test_unit_cute_layout
-  cutlass_test_unit_cute_core
-  cutlass_test_unit_cute_volta
-  cutlass_test_unit_cute_ampere
-  cutlass_test_unit_cute_hopper
-  cutlass_test_unit_cute_msvc_compilation
+
+if (NOT CUTLASS_ENABLE_SYCL)
```

Comment:
Can we change this if condition to add `sycl` specific tests first and then the cuda ones like we have done in the rest of the library?